### PR TITLE
Camp data pull

### PR DESCRIPTION
**Pull request by alvizek**
Updates made on this branch:

- edits the CAMP data pulls for Feather, Knights and Tisdale and streamlines the workflow (.Rmd sources either PC or Mac query)
- checks through the tables and fields for each CAMP db to make sure variables that are included/excluded make sense
- compares the proportion NA for each table/field across streams
- starts building the "data format" for CAMP users for ongoing data sharing

### CODE REVIEW

File: data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd [hunk 1]
```diff
@@ -0,0 +1,391 @@
+---
```
Comment (ErinCain) [inline data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd:1]: A lot of the tables generated in the Rmd are super long. The HTML does not seem that useful to look at. If we want it to be a more useful documentation resource we should update the display formating. 

I think fine for now but something we could update in the furture

Comment (ErinCain) [reply data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd:1]: same comment for tisdale & feather

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 2]
```diff
@@ -108,16 +113,16 @@ battle_trap_clean %>% glimpse
 - sub_week: is related to efficiency and included in efficiency table
 - sample_weight: same as cone variable
 - flow_start_meter, flow_end_meter, flow_set_time, velocity, turbidity,
-weather_code, lunar_phase, diel included in environmental table
+weather_code, lunar_phase, diel, river_left_depth, river_center_depth, river_right_depth included in environmental table
```
Comment (ErinCain) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:116]: I agree that it is fine to remove these but they are not all included in the environmental table. I think that is okay but we should make the comment consistent. Same for other tribs where these were removed.

**Review by ErinCain — APPROVED** [review]
All looks really good. I ran all the scripts that I was able to on my computer and they worked. I added a couple of comments for you to review and approved it so merge when you have looked through the comments.

File: data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd (hunk 1 above)
Comment (alvizek) [reply data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd:1]: I agree with you. i made some minor edits that at least make the html easier to read. the goal is to see the output tables pushed to google cloud. for future we can consider ways to make this better.

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 2 above)
Comment (alvizek) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd]: good catch. i revised comments for accuracy. next step is to revisit the standard formats. during this process i will make sure that the depth variables are included in the environmental table or clearly documented why they are left out.

**Pull request merged**